allow import of group and instance issues

cbergmann opened the pull request:
# Description

This allows searching and importing of gitlab issues on a group and instance level.

## Issues Resolved

#2529 

## Check List

- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

johannesjo commented:
Hey there! Could you check the linting issue?

Also: There was something similar implemented in the past, which broke the gitlab integration for a lot of people . Did you test the feature for possible side effects for different configurations?

cbergmann commented:
Hi,
thanks for looking over this.

The last commit fixes the linting issue.

I tested this with a Installation that was created before the patch and it seemd to be working. Then I changed the new setting and got results from other gitlab projects. I also changed it back to the default and only got the issues from the project. Any hints on what additional checks I could make are greatly welcome. While testing I also found that the link to the issue in the search result was broken and fixed that.

I think the last change that also broke it for some people was by me and you are rightfully critical of this change. But because of the new structure of the gitlab implementation this cange is a lot smaller and I think therefore better to review.

The only thing I am not sure about are automated tests. I have not yet written tests for angular.

johannesjo commented:
@cbergmann ping!
